pyproject.toml

```diff
@@ -15,27 +15,25 @@ line-length = 120
 target_version = ['py39']
 include = '\.py$'
 
-[tool.ruff.isort]
-known-third-party = ["cognite.client"]
-
 [tool.ruff]
-# See https://beta.ruff.rs/docs/rules for an overview of ruff rules
 line-length = 120
 target-version = "py39"
-exclude = ["examples", "scripts", "cognite/powerops/client/_generated"]
-select = ["E","W","F","I","RUF","TID","UP", "B", "FLY", "PTH", "ERA"]
-fixable = ["E","W","F","I","RUF","TID","UP", "B", "FLY", "PTH", "ERA"]
+exclude = ["scripts", "cognite/powerops/client/_generated"]
 fix = true
+
+[tool.ruff.lint]
 ignore = [
     # This is causing issues for pydantic which does not support use of X | Y for type annotations before 3.10
     "UP007", # Use X | Y for type annotations
     # Typer do function calls when setting up the CLI.
     "B008", # Do not perform function call {name} in argument defaults,
     "RUF012", # Insistance on using ClassVar to mark class variables
 ]
+select = ["E","W","F","I","RUF","TID","UP", "B", "FLY", "PTH", "ERA"]
+fixable = ["ALL"]
```

Comment:
@nodegard is this intentional ? to do ALL ?

Reply:
"ALL" uses what's in `select`, so "ALL" is the same as the previous configuration since select and fixable were using the same list. Just easier to maintain if you add or remove things from `select` 👍🏻